interfaces/Input_to_core.atd

```diff
@@ -42,3 +42,18 @@ type target = {
  * file with both PL and generic/regexp rules.
  *)
 type targets = target list
+
+
+(*****************************************************************************)
+(* Synthesizing from diffs *)
+(*****************************************************************************)
+
+type diff_file = {
+  filename : string;
+  diffs : string (* start_line-end_line *) list;
+  url : string
+}
```

Comment:
* why not `(int * int) list` for (start_line, end_line)?
* is line-numbering 0-based or 1-based?
* what's the `url` field?

Reply:
Thanks for calling out this and all the other places where I didn't document!